WRP-1058: Fixed enact test `--watch` option is not working with the latest `jest-watch-typeahead` (#314)
enactjs/cli: CHANGELOG.md

## unreleased

* Upgraded `eslint-plugin-react` version to `^7.32.2`.
* Removed `v8-compile-cache` dependency to fix problems related to ES modules.

### test

> * Updated `jest-watch-typeahead` to `2.2.2`.

Reviewer: Let's not log this.

> * Removed `v8-compile-cache` dependency to fix problems related to ES modules.
> 
> ### test
> 
> * Updated `jest-watch-typeahead` to `2.2.2`.

Reviewer: ```suggestion

### test

* Fixed `--watch` option is not working by removing `v8-compile-cache` module.
```

* Upgraded `eslint-plugin-react` version to `^7.32.2`.

### test

> * Fixed `--watch` option is not working by removing `v8-compile-cache` module.

Reviewer: I'm sorry.
```suggestion
* Fixed `--watch` option is not working with the latest `jest-watch-typeahead` module.
```
